src/join.sol

```diff
@@ -70,7 +70,7 @@ contract GemJoin is DSNote {
     bytes32 public ilk;
     GemLike public gem;
     uint    public dec;
-    uint256 public live;  // Access Flag
+    uint    public live;  // Access Flag
```

Comment:
better to make both variables `uint256` for those extra code alignment points

Reply:
It is tempting but then I'd feel obligated to suffixify with 256 throughout the file...

Reply:
Yeah alignment with other words like `address/mapping/GemLIke/VatLike/etc` and explicitness lead me to use `uint256` instead of `uint`. I guess we can have a discussion later and create a PR to standardise it properly in the whole repo.

Reply:
Yeah doing it piecemeal is inefficient. At some point we should actually establish strict and explicit coding conventions but I don't view that as urgent.